src/vision/cfg/launch_params_out.yaml

```diff
@@ -0,0 +1,6 @@
+h_high: 318
```

Comment:
Hmm, I'm a bit confused about the name of this file. A description in the file might clear things up a bit, but is there a more descriptive name we could use?

Reply:
This file is saved by ros's dynamic reconfigure package, if we add a comment ros will just overwrite it when it's saved.

Reply:
Renamed